processing/src/test/java/io/druid/segment/filter/BoundFilterTest.java

```diff
@@ -77,10 +77,10 @@ public BoundFilterTest(
   public void testLexicographicMatchEverything()
```

Comment:
Could you add a test somewhere in this file for a filter with extraction functions? Also for SelectorFilterTest and InFilterTest.